src/components/getFromApi/GetReviewListOfHotel.jsx

```diff
@@ -0,0 +1,12 @@
+import axios from './AxiosConfig'
+
+////id - of object 
+//// loc -location 
+const GetReviewsListOfHotel = (args) =>{
```

Comment:
You can use 
= ({ id }) => 
Let me know if you have any questions.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment